subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -193,74 +499,119 @@ static uint32_t init(uint8_t chan, uint8_t phy, void (*isr)(void *))
 	/* NOTE: No whitening in test mode. */
 	radio_phy_set(test_phy, test_phy_flags);
 	radio_tmr_tifs_set(150);
-	radio_tx_power_max_set();
+
+	err = tx_power_set(tx_power);
```

Comment:
declare a local `uint8_t ret;`

```suggestion
	ret = tx_power_set(tx_power);
```